railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -130,6 +130,9 @@ def config_when_updating
       assets_config_exist            = File.exist?("config/initializers/assets.rb")
       csp_config_exist               = File.exist?("config/initializers/content_security_policy.rb")
 
+      load_defaults = File.read("config/application.rb").match(/config\.load_defaults (\d\.\d)/)
```

Comment:
I think we also need a test that this matches the generated content (e.g., right now it would stop working for version 10.0, but no test would fail). Do you have any ideas how we could do that?

Reply:
Two other thoughts on the regex:

1. Start with `/^\s*`, to ignore commented lines.
2. Technically, the argument is allowed to be a string (because `6.10.to_s` and `"6.10".to_s` are very different)... but maybe that's not worth worrying about for now.